Samples/DirectMLNpuInference/main.cpp

```diff
@@ -72,7 +110,8 @@ void InitializeDirectML(ID3D12Device1** d3dDeviceOut, ID3D12CommandQueue** comma
                 );
             if (d3d12CreateDevice)
             {
-                THROW_IF_FAILED(d3d12CreateDevice(adapter.Get(), D3D_FEATURE_LEVEL_1_0_CORE, IID_PPV_ARGS(&d3dDevice)));
+                // The GENERIC feature level minimum allows for the creation of both compute only and generic ML devices.
+                THROW_IF_FAILED(d3d12CreateDevice(adapter.Get(), D3D_FEATURE_LEVEL_1_0_GENERIC, IID_PPV_ARGS(&d3dDevice)));
```

Comment:
GENERIC only exists in Windows 24H2 and newer, so this sample won't work on older versions of Windows. The most robust logic would check both (see https://github.com/microsoft/DirectML/blob/7138c1c00d5ea1d0d55ff6a07e12b48f440907ca/Samples/DirectML_ESRGAN/helpers.cpp#L7-L101).

Reply:
I see. Updated so that in the case there are no GENERIC device available, we create with the CORE feature level instead. Let me know if the new logic looks good!

Reply:
> Let me know if the new logic looks good!

Hmm, I'm not seeing any new logic. Did you push the changes?

Reply:
Ah. Fixed.